src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -19,7 +19,7 @@ def put(self, request, project, filter_id):
 
         """
         current_filter = None
-        for flt in message_filters.get_all_filters():
+        for flt in message_filters.get_all_filter_specs():
             if flt.spec.id == filter_id:
```

Comment:
```suggestion
            if flt.id == filter_id:
```

And same bug below:


https://github.com/getsentry/sentry/blob/a974b513b3b3be119a594f5e71db0adfce1d81ef/src/sentry/api/endpoints/project_filter_details.py#L29